api/views/phones.py

```diff
@@ -513,14 +522,44 @@ def _handle_sms_reply(relay_number, real_phone, inbound_body):
         body=inbound_body,
         to=last_text_sender.inbound_number,
     )
+    relay_number.remaining_texts -= 1
+    relay_number.texts_forwarded += 1
+    relay_number.save()
 
 
 def _check_disabled(relay_number, contact_type):
     # Check if RelayNumber is disabled
     if not relay_number.enabled:
+        attr = f"{contact_type}_blocked"
+        setattr(relay_number, attr, getattr(relay_number, attr) + 1)
+        relay_number.save()
         raise exceptions.ValidationError(f"Number is not accepting {contact_type}s.")
 
 
+def _check_remaining(relay_number, resource_type):
```

Comment:
This is basically the logic here:
![image](https://user-images.githubusercontent.com/71928/188975844-a35dce4d-8bec-43ae-ac96-3ff259cb245c.png)

Reply:
Oh, except step 1.5 in the top right: set last-checked date.

Reply:
The top of this is moved into the new `update_phone_remaining_stats.py` command.